### DISCUSSION TITLE
Timeouts connecting to git.github.com

### DISCUSSION THREAD

**Discussion by jayhill90** [community/community#21913] [New to GitHub]
<p>I was trying to push some new code that I’ve worked the past few days on to Github and I haven’t had any issues pushing any changes up. I’ve ensured that my ~/.ssh/config file is good as well but I keep getting a timeout.</p>
<pre><code>ssh -vvT git@github.com
OpenSSH_7.6p1, LibreSSL 2.6.2
debug1: Reading configuration data /Users/jay.hill/.ssh/config
debug1: /Users/jay.hill/.ssh/config line 3: Applying options for *
debug1: /Users/jay.hill/.ssh/config line 9: Applying options for github.com
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 48: Applying options for *
debug2: ssh_connect_direct: needpriv 0
debug1: Connecting to git.github.com port 22.
ssh: connect to host git.github.com port 22: Operation timed out
</code></pre>
<p>Anyone know what I might be doing wrong here? It worked like two weeks ago just fine. </p>

**Comment by JoeWindowsNoQ** [comment, marked as answer]
<p>Is it related to this?</p>
<p><a href="https://github.com/blog/2507-weak-cryptographic-standards-removed" rel="nofollow noopener">https://github.com/blog/2507-weak-cryptographic-standards-removed</a></p>

**Comment by jayhill90** [comment]
<p>I think it might have been. I generated a new SSH key, which it was probably time to cycle anyways, and was able to connect once I setup an SSH config to conenct to <a href="http://ssh.github.com" rel="nofollow noopener">ssh.github.com</a> with my key. </p>
